docsite/rst/playbooks_lookups.rst

```diff
@@ -347,6 +347,96 @@ TLSA         usage, selector, mtype, cert
 TXT          strings
 ==========   =============================================================================
 
+.. _mongodb_lookup:
+
+MongoDB Lookup
+``````````````
+.. versionadded:: 2.1.1
```

Comment:
I know. But since I don't have a crystal ball, I don't know if I leave this blank, put a placeholder, be optimistic and put whatever is in production + 1 or anything like that.
That's what I meant.

I hope 2.1.1 is ok.